src/player.cpp

```diff
@@ -3720,9 +3720,13 @@ bool player::consume_item( item &target )
     return false;
 }
 
-bool player::consume( int target_position )
+bool player::consume( item_location &loc )
```

Comment:
Stupid question: why does this require a non-const reference? Does the original (a simple value) not work? I don't see `loc` being changed within the function.

Reply:
because player::consume_item() below on line 3730 takes a non-const item &

Reply:
A `item_location` (not reference, just a value) will also provide a non-const item. The reference on the parameter restricts callers needlessly.

Reply:
all right, neat. thanks

Reply:
![image](https://user-images.githubusercontent.com/30208401/69480661-8644b180-0dd7-11ea-9e34-7d77d7fd296d.png)
hmmm, are you sure about that?

Reply:
my bad, i just realized what you were saying.